prs/prs-middleware/src/main/java/com/adorsys/webank/security/CachingRequestBodyWrapper.java

```diff
@@ -0,0 +1,66 @@
+package com.adorsys.webank.security;
+
+import jakarta.servlet.ReadListener;
+import jakarta.servlet.ServletInputStream;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletRequestWrapper;
+import java.io.BufferedReader;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class CachingRequestBodyWrapper extends HttpServletRequestWrapper {
```

Comment:
CachingRequestBodyWrapper, is it necessary? If you do without, what will happen?